docs/developer-guide/implem-crud.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
same feedback about how we tag metadata front matter, we'll figure it out, but for now we can use the existing model